src/utils/pluginDriver.ts

```diff
@@ -104,7 +104,9 @@ export function createPluginDriver(
 			},
 			parse: graph.contextParse,
 			resolveId(id: string, parent: string) {
-				return pluginDriver.hookFirst('resolveId', [id, parent]);
+				return this.isExternal(id, parent, false)
+					? false
+					: pluginDriver.hookFirst('resolveId', [id, parent]);
```

Comment:
This logic is now duplicated with https://github.com/rollup/rollup/blob/master/src/Graph.ts#L783.

I would prefer if we could keep the separation for all calls the same I think.